docs/intro/tutorial03.txt

```diff
@@ -83,19 +83,19 @@ Wire these new views into the ``polls.urls`` module by adding the following
 .. snippet::
     :filename: polls/urls.py
 
-    from django.conf.urls import url
+    from django.urls import path
 
     from . import views
 
     urlpatterns = [
         # ex: /polls/
-        url(r'^$', views.index, name='index'),
+        path('', views.index, name='index'),
         # ex: /polls/5/
-        url(r'^(?P<question_id>[0-9]+)/$', views.detail, name='detail'),
+        path('<int:question_id>/', views.detail, name='detail'),
         # ex: /polls/5/results/
-        url(r'^(?P<question_id>[0-9]+)/results/$', views.results, name='results'),
+        path('<int:question_id>/results/', views.results, name='results'),
         # ex: /polls/5/vote/
-        url(r'^(?P<question_id>[0-9]+)/vote/$', views.vote, name='vote'),
+        path('<int:question_id>/vote/', views.vote, name='vote'),
     ]
 
 Take a look in your browser, at "/polls/34/". It'll run the ``detail()``
```

Comment:
The unchanged segment after this needs changes:

```
'^polls/' should become 'polls/'
r'^(?P<question_id>[0-9]+)/$' should become '<int:question_id>/'
question_id='34' should become question_id=34 (because it is now actually casted to an integer!)
```

also should be some mention of the "<int:...>" syntax, as well as converters.